src/layers/models/layersManager.ts

```diff
@@ -146,7 +146,7 @@ class LayersManager {
 
   public async updateLayer(layerName: string, layerRequest: ILayerPostRequest): Promise<void> {
     this.logger.info(`Update layer: '${layerName}' request`);
-    const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, layerRequest.format);
+    const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, this.mapToTileFormat(layerRequest.format));
```

Comment:
separate to a different row "this.mapToTileFormat(layerRequest.format)", not as parameter inside other function

Reply:
done